src/Common/tests/System.Net/Sockets/Fletcher32.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Net.Sockets.Tests
+{
+    // Utility type for calculating a running Fletcher-32 checksum.
+    internal struct Fletcher32
```

Comment:
Seems like overkill. Why not just send a recognizable byte pattern instead (e.g. always the string ABCDEF...)?